app/Livewire/Notifications/Pushover.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace App\Livewire\Notifications;
+
+use App\Models\PushoverNotificationSettings;
+use App\Models\Team;
+use App\Notifications\Test;
+use Livewire\Attributes\Validate;
+use Livewire\Component;
+
+class Pushover extends Component
+{
+    public Team $team;
+
+    public PushoverNotificationSettings $settings;
+
+    #[Validate(['boolean'])]
+    public bool $pushoverEnabled = false;
+
+    #[Validate(['string', 'nullable'])]
+    public ?string $pushoverToken = null;
+    
+    #[Validate(['string', 'nullable'])]
+    public ?string $pushoverUser = null;
+
+    #[Validate(['boolean'])]
+    public bool $deploymentSuccessPushoverNotifications = false;
+
+    #[Validate(['boolean'])]
+    public bool $deploymentFailurePushoverNotifications = true;
+
+    #[Validate(['boolean'])]
+    public bool $statusChangePushoverNotifications = false;
+
+    #[Validate(['boolean'])]
+    public bool $backupSuccessPushoverNotifications = false;
+
+    #[Validate(['boolean'])]
+    public bool $backupFailurePushoverNotifications = true;
+
+    #[Validate(['boolean'])]
+    public bool $scheduledTaskSuccessPushoverNotifications = false;
+
+    #[Validate(['boolean'])]
+    public bool $scheduledTaskFailurePushoverNotifications = true;
+
+    #[Validate(['boolean'])]
+    public bool $dockerCleanupSuccessPushoverNotifications = false;
+
+    #[Validate(['boolean'])]
+    public bool $dockerCleanupFailurePushoverNotifications = true;
+
+    #[Validate(['boolean'])]
+    public bool $serverDiskUsagePushoverNotifications = true;
+
+    #[Validate(['boolean'])]
+    public bool $serverReachablePushoverNotifications = false;
+
+    #[Validate(['boolean'])]
+    public bool $serverUnreachablePushoverNotifications = true;
+
+
+    public function mount()
+    {
+        try {
+            $this->team = auth()->user()->currentTeam();
+            $this->settings = $this->team->pushoverNotificationSettings();
```

Comment:
This line can not have `()`. it needs to be:
`$this->settings = $this->team->pushoverNotificationSettings;`

Reply:
@zaosoula Found it (at least this should be the issue).

Reply:
Perfect thank you
I will proceed with the testing

Reply:
It's working
Thanks for the help
![IMG_7234](https://github.com/user-attachments/assets/d0d3f68c-632f-4169-ba91-41d64ec4ff32)